# Automatically Generated in 2017-06-12 22:33:47.452947
# MESSAGE: ODOMETER_DELTA_RAW
# Description:Measurement of Odometry Sensor within a time-period (raw ticks)
uint8 ID = 139
uint8 sysid
uint8 compid
uint32 timestamp
float32 delta
int32 xdist
int32 ydist
int32 zdist
int16 quality
